docs/csdk-guide.md

```diff
@@ -29,6 +29,32 @@ code and without CarbonSession.
 
 In the carbon jars package, there exist a carbondata-sdk.jar, 
 including SDK reader for C++ SDK.
+
+##Compile/Build CSDK
+CSDK supports cmake based compilation and has dependency list in CMakeLists.txt.
+<br> Prerequisites
+<br>GCC >=4.8.5
+<br>Cmake >3.13
+<br>Make >=4.1
+
+Steps 
+1. Go to CSDK folder(/opt/.../CSDK/) 
+2. Create build folder . (/opt/.../CSDK/build) 
+3. Run Command from build folder `cmake ../`
+4. `make`
```

Comment:
How to run it by command?

Reply:
cmake,make are to be executed in terminal.

Reply:
I mean how to run code after make?

Reply:
It is same like before not change.  After `make` command you will get the executable program (named CSDK). so directly execute the same .   ./CSDK . if Result to be redirected to xml then use command like  ./CSDK --gtest_output="xml:${REPORT_PATH}/CSDK_Report.xml".